apps/stripe/vitest.config.ts

```diff
@@ -25,6 +25,19 @@ export default defineConfig({
           setupFiles: "./src/__tests__/setup.integration.ts",
```

Comment:
do we need old integration at all? I think we can move manifest test into units

Reply:
ok